.changeset/tough-olives-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-docs/writing-style': patch
```

Comment:
I think this is minor. It's not just fixing things but actually improving the behavior.